presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -309,6 +282,12 @@ public CostAssertionBuilder memoryUnknown()
             return this;
         }
 
+        public CostAssertionBuilder hasUnknownComponents()
```

Comment:
rename to isUnknown since we don't allow unknown components anymore  (also for the method in PlanNodeCostEstimate).